repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/framework/mocks/MockIdentityV2Service.groovy

```diff
@@ -469,8 +485,10 @@ class MockIdentityV2Service {
             // THEN return the Admin token response.
             if (request.body.contains("username") &&
                     request.body.contains(client_username) &&
-                    request.body.contains("apiKey") &&
-                    request.body.contains(client_apikey)) {
+                    ((request.body.contains("apiKey") &&
```

Comment:
Interestingly, IntelliJ's auto-formatter keeps this formatting. I think the reason is that the logic is kind of complex, so parenthesis are being used to group predicates so that the connectives fit together in the way we want them to.

*tldr;* Parenthesis for excepting operator precedence lead to odd auto-formatting.